Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.6
+// swift-tools-version:5.7.1
```

Comment:
I don't think it should be visible, but we should still note in the release notes.

Reply:
Agreed, added a changelog entry. Let me know if you can think of a better description. Hopefully it conveys that it should be a no-op change.